Geometry/MuonNumbering/src/DTNumberingScheme.cc

```diff
@@ -32,7 +42,8 @@ int DTNumberingScheme::baseNumberToUnitNumber(const MuonBaseNumber& num) const {
     edm::LogVerbatim("DTNumberingScheme") << level << " " << num.getSuperNo(level) << " " << num.getBaseNo(level);
   }
 #endif
-  if (num.getLevels() != theWireLevel) {
+  //  if (num.getLevels() != theWireLevel) {
+  if (num.getLevels() < theStationLevel) {
```

Comment:
Are you sure you want to keep this version of the `if` statement? What is the correct number of levels? Have you talked to Sunanda about this? He recently fixed a similar problem with the GEMs.
Whatever you decide, make sure the message on line 48 matches the `if`.

Reply:
@cvuosalo I didn't talk with @bsunanda 

>>He recently fixed a similar problem with the GEMs.

@bsunanda how can you fix it for GEMs?

Reply:
@cvuosalo I fixed line 48

Reply:
@cvuosalo probably it could be better to go forward with this PR in order to test it (i.e "please test") in order to see if all is fine, then I will get in touch (offline) with @bsunanda and you in order to understand better the number levels issue

Reply:
@slomeo I just give a warning sign and proceed. I test the number of levels at each step before assigning chamber, roll, ...